packages/e2e-test-utils-playwright/src/request-utils/themes.ts

```diff
@@ -13,12 +13,25 @@ async function activateTheme(
 	let response = await this.request.get( THEMES_URL );
 	const html = await response.text();
 	const optionalFolder = '([a-z0-9-]+%2F)?';
-	const matchGroup = html.match(
-		`action=activate&amp;stylesheet=${ optionalFolder }${ encodeURIComponent(
+
+	// The `optionalFolder` regex part matches paths with a folder,
+	// so it will return the first match, which might contain a folder.
+	// First try to honor the including theme slug, that is, without a folder.
+	let matchGroup = html.match(
+		`action=activate&amp;stylesheet=${ encodeURIComponent(
```

Comment:
The first match is to honor any incoming `themeSlug` when the tester wants to be explicit.